portfolio/src/main/webapp/script.js

```diff
@@ -55,18 +55,23 @@ function rotateImages() {
 }
 
 function getCommentsUsingArrows() {
```

Comment:
Is this function name accurate? It's unclear to me what "usingArrows" means--it looks like you're simply waiting for the page to load, then fetching all the comments and inserting them into the document body.

Reply:
In my original version I had tested two versions -- arrows and async/await but I ended up going with arrows (=>) but you're right, it needs a more clear name now that there aren't two functions doing the same thing!

Reply:
I see! I would say that good function names describe what they do, not necessarily how they do it. Here I might go with `fetchAndDisplayComments()`. The fact that it's written as an arrow function is an implementation detail that does not (to me) seem interesting to the function's callers.